### ISSUE TITLE
Null in Java SDK Causes Application Crash

### ISSUE THREAD

**Issue by baonguyen-bm** [flipt-io/flipt-client-sdks#573]
Hello team,

I've provided an example of Java code using the Java SDK:

```java
public static void main(String[] args) {
    FliptEvaluationClient fliptClient = null;
    try {
        fliptClient = FliptEvaluationClient.builder()
            .url("https://api.feature-flag.test.mfw.work")
            .authentication(new ClientTokenAuthentication("Ig4MkhZzbnXMhhKQpHA5HoTpKU_L3HLXBenMid6L-Bc="))
            .build();

        Map<String, String> context = new HashMap<>();
        context.put("fizz", "buzz");

        VariantEvaluationResponse response = fliptClient.evaluateVariant("flag1", null, context);
    } catch (Exception e) {
        e.printStackTrace();
    } finally {
        // Important: always close the client to release resources
        if (fliptClient != null) {
            fliptClient.close();
        }
    }
}
```

As demonstrated, I passed a `null` value to `fliptClient.evaluateVariant`, resulting in the following error:

```
thread '<unnamed>' panicked at flipt-engine-ffi/src/lib.rs:344:90:
called `Result::unwrap()` on an `Err` value: Error("invalid type: null, expected a string", line: 1, column: 36)
stack backtrace:
   0:        0x122157578 - <unknown>
   1:        0x12219a9e8 - <unknown>
   2:        0x12214c9fc - <unknown>
   3:        0x122159b38 - <unknown>
   4:        0x1221596e8 - <unknown>
   5:        0x12215a6cc - <unknown>
   6:        0x12215a0f8 - <unknown>
   7:        0x122157a3c - <unknown>
   8:        0x122159dc0 - <unknown>
   9:        0x1221cd630 - <unknown>
  10:        0x1221cda4c - <unknown>
  11:        0x121ed85e0 - <unknown>
  12:        0x121ed7c38 - <unknown>
  13:        0x10143804c - <unknown>
thread '<unnamed>' panicked at core/src/panicking.rs:221:5:
panic in a function that cannot unwind
stack backtrace:
   0:        0x122157578 - <unknown>
   1:        0x12219a9e8 - <unknown>
   2:        0x12214c9fc - <unknown>
   3:        0x122159b38 - <unknown>
   4:        0x1221596e8 - <unknown>
   5:        0x12215a6cc - <unknown>
   6:        0x12215a0d0 - <unknown>
   7:        0x122157a3c - <unknown>
   8:        0x122159dc0 - <unknown>
   9:        0x1221cd660 - <unknown>
  10:        0x1221cd6d8 - <unknown>
  11:        0x1221cd850 - <unknown>
  12:        0x121ed7d64 - <unknown>
  13:        0x10143804c - <unknown>
thread caused non-unwinding panic. aborting.

Process finished with exit code 134 (interrupted by signal 6:SIGABRT)
```

I understand that Rust does not have a `null` value, but in this scenario, I expected an exception or some form of null handling. Unfortunately, my application crashed. I have created an issue on GitHub to address this, and I kindly request an update to the convention and language to handle such cases more gracefully.

**Comment by markphelps** [comment]
@baonguyen-bm thanks for reporting. I will create a new release now of the java sdks (and others) with this fixed

**Comment by markphelps** [comment]
@baonguyen-bm I have just released new versions of the Java SDK that should resolve this. please let me know if this fixes your issue after upgraded

**Comment by baonguyen-bm** [comment]
I got it. The issue have been fixed after upgrading to 0.12.0

Thanks @markphelps
